### PR TITLE
DOM Manipulation and Events: Add aditional resource

### PR DESCRIPTION
**Pull request by javier-rosales**
## This PR
- Adds a new addtional resource for the topic "Events", this covers material that until now was only available on videos in the lesson.

## Issue

Closes #XXXXX

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: foundations/javascript_basics/DOM_manipulation_and_events.md [hunk 1]
```diff
@@ -422,3 +422,4 @@ This section contains helpful links to related content. It isn’t required, so
 *   This [W3Schools](https://www.w3schools.com/js/js_htmldom.asp) article offers simple and easy-to-understand lessons on the DOM.
 *   [JS DOM Crash Course](https://www.youtube.com/watch?v=0ik6X4DJKCc&list=PLillGF-RfqbYE6Ik_EuXA2iZFcE082B3s) is an extensive and well explained 4 part video series on the DOM by Traversy Media.
 *   [Understanding The Dom](https://www.digitalocean.com/community/tutorial_series/understanding-the-dom-document-object-model) is an aptly named article-based tutorial series by DigitalOcean.
+*   [Introduction to events - MDN](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Events) covers the same topics you learned in this lesson on *events* (videos included).
```
Comment (rlmoser99) [inline foundations/javascript_basics/DOM_manipulation_and_events.md:425]: I think the "(videos included)" is a little confusing and does add not much value to this sentence. In addition, the use of italics is not needed I prefer moving MDN out of the link. So something like this:

```suggestion
*   [Introduction to events](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Events) by MDN covers the same topics you learned in this lesson on events.
```

**Review by rlmoser99 — CHANGES_REQUESTED** [review]
This addition will need a few small changes. Feel free to alter my solution, based on 3 small things that I would like to see changed.

**Review by rlmoser99 — APPROVED** [review]
Thank you for makes these suggestions so quickly!

**Comment by javier-rosales** [comment]
Thank you for supporting this awesome platform!

**Pull request merged**